cg/cli/workflow/mip/base.py

```diff
@@ -44,15 +47,16 @@ def config_case(context: click.Context, case_id: str, panel_bed: str, dry_run: b
 
 @click.command()
 @OPTION_DRY
+@OPTION_DNA_PANEL_GENOME_BUILD
 @ARGUMENT_CASE_ID
 @click.pass_context
-def panel(context: click.Context, case_id: str, dry_run: bool):
-    """Write aggregated gene panel file"""
+def panel(context: click.Context, case_id: str, dry_run: bool, panel_genome_build: str = None):
+    """Write aggregated gene panel file exported from Scout"""
 
     analysis_api: MipAnalysisAPI = context.obj["analysis_api"]
     analysis_api.verify_case_id_in_statusdb(case_id=case_id)
 
-    bed_lines: List[str] = analysis_api.panel(case_id=case_id)
+    bed_lines: List[str] = analysis_api.panel(case_id=case_id, genome_build=panel_genome_build)
```

Comment:
I suggest maybe to edit the analysis_api.panel method with each default genome build instead?Such that analysis_api.panel has different defaults as RNA and DNA api versions. Then the panel_genome build can be optional and defaults are set depending on which class is being used.

If we do this, the panel cli method will be generic and can be reused and improted by rna and dna cli the same way

Reply:
This will actually allow us to move mip dna start as common command to be imported in rna/dna context and it can be adapted depending on workflow

Reply:
Done